admin/create-theme/theme-zip.php

```diff
@@ -47,14 +47,15 @@ public static function add_activated_fonts_to_zip( $zip, $theme_json_string ) {
 			}
 			$font_slugs_to_remove[] = $font_family['slug'];
 			foreach ( $font_family['fontFace'] as &$font_face ) {
-				$font_filename = basename( $font_face['src'] );
-				$font_dir      = wp_get_font_dir();
+				$font_filename  = basename( $font_face['src'] );
+				$font_dir       = wp_get_font_dir();
+				$font_face_path = path_join( $theme_font_asset_location, $font_filename );
 				if ( str_contains( $font_face['src'], $font_dir['url'] ) ) {
-					$zip->addFileToTheme( $font_dir['path'] . '/' . $font_filename, $theme_font_asset_location . '/' . $font_filename );
+					$zip->addFileToTheme( $font_dir['path'] . '/' . $font_filename, $font_face_path );
```

Comment:
Could we also use `path_join` here?

```suggestion
					$zip->addFileToTheme( path_join( $font_dir['path'], $font_filename, $font_face_path ) );
```

Reply:
The `path_join()` function [only accepts two arguments](https://github.com/WordPress/wordpress-develop/blob/66b5d25be2b5e69833bacfe8f64de660ffc2bfa9/src/wp-includes/functions.php#L2155), so I think it's like this

```php
$zip->addFileToTheme( path_join( $font_dir['path'], $font_filename ), $font_face_path );
```

Reply:
Thanks for catching that, got the argument list wrong on the suggestion — edited

Reply:
I tested with your suggestion @vcanales and that works as expected.  I'm going to commit that suggestion and bring this one in.

Reply:
:+1: added